tests/qchem/test_factorization.py

```diff
@@ -719,3 +719,49 @@ def test_chemist_transform(
     )
 
     assert np.allclose(one_body_corr, one_body_correction)
+
+
+@pytest.mark.parametrize(
+    ("core_shifted", "one_body_shifted", "two_body_shifted"),
+    [
+        # (Build HeH+ Molecule and Chemist Hamiltonian)
+        # H_chemist = qml.jordan_wigner(
+        #   fermionic_observable(core_shifted, one_body_shifted, 2 * two_body_shifted, chemist=True)
+        # ).matrix()
+        # eigvals, eigvecs = np.linalg.eigh(H_chemist)
+        # for eigval, eigvec in zip(eigvals, eigvecs.T):
+        #     if (eigvec @ qml.matrix(qml.qchem.particle_number(2 * mol.n_orbitals)) @ eigvec.conj().T) == mol.n_electrons:
+        #         print(eigval) # --> -2.688647053431185
+        #         break
+        (
+            np.array([0.14782753]),
```

Comment:
How the reference values are computed? Does the comment above try to validate the H computed with the shifted values? If yes, please explain in 1-2 lines.